### PR TITLE
Add login component

### PR DESCRIPTION
**Pull request by charmata**

### CODE REVIEW

File: client/src/components/Login/index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  handleChange = e => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value
+    });
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
```
Comment (dvadz) [inline client/src/components/Login/index.js:18]: @charmata destructure the email and password from state

Comment (dvadz) [reply client/src/components/Login/index.js:18]: ` const {email: username, password} = this.state `

File: client/src/components/Login/index.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        console.log("Logged in");
+        this.setState({ email: "", password: "" });
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  };
+
+  render() {
+    return (
```
Comment (dvadz) [inline client/src/components/Login/index.js:34]: @charmata please destructure the sate and props from the render return into its variable scope

**Review by malzoibi — APPROVED** [review]

**Review by dvadz — CHANGES_REQUESTED** [review]
@charmata couple of pr comments

**Pull request merged**